proto/buf/alpha/registry/v1alpha1/token.proto

```diff
@@ -48,6 +48,9 @@ message CreateTokenRequest {
   // The time until which the token should be valid.
   // Must be in the future. May be null for no expiry.
   google.protobuf.Timestamp expire_time = 2;
+  // The ID of the user to create a new token, can only be
+  // the user who perform this action, or a machine user.
+  string user_id = 3;
```

Comment:
will this default to the user making the request, if empty? do we want to mention that in the comment?

Reply:
I think in the end we will want all clients to provide this field, but updated as empty should be accepted for now